recipe-server/normandy/settings.py

```diff
@@ -380,6 +385,9 @@ class Development(Base):
     SWAGGER_SETTINGS = Base.SWAGGER_SETTINGS
     SWAGGER_SETTINGS['VALIDATOR_URL'] = None
 
+    # URL that hosts Experimenter experiment definitions
+    EXPERIMENTER_PENDING_URL = values.URLValue('https://localhost/api/v1/experiments/?status=Pending')
```

Comment:
Separately, I think this might be better as `EXPERIMENTER_API_URL` that points to `/api/v1` instead of pointing to a very specific API endpoint. the frontend can build the more specific URL from the less specific one, but it isn't very sound to go backwards, if we want to in the future.